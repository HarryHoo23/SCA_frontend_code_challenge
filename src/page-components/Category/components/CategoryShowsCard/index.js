const CategoryShowsCard = () => {
  return (
    <div>
      This is a card.
    </div>
  )
}

export default CategoryShowsCard;
