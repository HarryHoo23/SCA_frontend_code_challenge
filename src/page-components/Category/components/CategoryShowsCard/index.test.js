import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import CategoryShowsCard from './index';

describe('CategoryShowsCard component', () => {
  test('match snapshot', () => {
    const card = {
      id: '6c94c7db-0616-4c10-8745-aa720058d8e8',
      slug: 'adam-carolla-show',
      name: 'Adam Carolla Show',
      description:
        'Welcome to the Adam Carolla Show! The new home forâ€¦y, Dana Gould, Doug Benson, and many, many more..',
      images: {
        squareLarge: {
          pixelWidth: 320,
          url: "https://listnr-img-prod.scalabs.com.au/api/assets/mercury/cbea158c-f84a-4904-8fae-220d4d2e91c7/v0?width=320&height=320"
        } 
      }
    };
  
    const tree = renderer.create(<CategoryShowsCard card={card} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});