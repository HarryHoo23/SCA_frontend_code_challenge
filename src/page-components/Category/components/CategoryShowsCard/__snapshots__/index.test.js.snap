// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="src__Box-sc-1sbtrzs-0 sc-jRQBWg jUjbhW cuPwvw"
  color="white"
  data-testid="card-6c94c7db-0616-4c10-8745-aa720058d8e8"
  width={
    Array [
      0.5,
      0.3333333333333333,
      0.25,
    ]
  }
>
  <img
    alt=""
    className="sc-bdvvtL eVTLAU sc-gKclnd daYdTy"
    height="auto"
    loading="lazy"
    src=""
    width="320px"
  />
  <h4
    className="sc-gsDKAQ sc-dkPtRN sc-eCImPb jGAxre kiuwfu iTqlBH"
    fontWeight="bold"
    opacity={null}
  >
    Adam Carolla Show
  </h4>
  <p
    className="sc-gsDKAQ sc-dkPtRN sc-eCImPb bRuDTs kiuwfu kQXgnl"
    fontWeight="normal"
    opacity={null}
  >
    Welcome to the Adam Carolla Show! The new home forâ€¦y, Dana Gould, Doug Benson, and many, many more..
  </p>
</div>
`;
